src/utils/update/review_trusts.ts

```diff
@@ -138,3 +138,13 @@ export const putTrustInChangeScenario = (appData: ApplicationData, trustId: stri
     };
   }
 };
+
+export const resetTrustInReviewPagesReviewed = (trustInReview: Trust) => {
```

Comment:
Function name is not that clear to me. Would 'setAllTrustsAsNotReviewed' accurately describe what this is doing and is it any better?

Reply:
it's only for the trust that is currently in review so not applicable to all trusts. Is resetPagesReviewedOnTrustInReview any better?

Reply:
`resetPagesReviewed` maybe?